Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -2823,10 +2822,23 @@ static NSStringCompareOptions compareOptionsForFindOptions(WebCore::FindOptions
         return nullptr;
 
     auto& context = buffer->context();
-    context.translate(-rectInContentsCoordinates.location());
-    context.scale(m_scaleFactor);
-    context.translate(centeringOffset());
-    paintPDFContent(context, rectInContentsCoordinates, PaintingBehavior::PageContentsOnly);
+
+    {
+        GraphicsContextStateSaver saver(context);
+
+        context.scale(m_scaleFactor);
+        context.translate(-rectInContentsCoordinates.location());
+
+        paintPDFContent(context, rectInContentsCoordinates, PaintingBehavior::PageContentsOnly);
+    }
+
+    // FIXME: Figure out how to share this with WebTextIndicatorLayer.
+#if PLATFORM(MAC)
+    Color highlightColor = *roundAndClampToSRGBALossy([NSColor findHighlightColor].CGColor);
+#else
+    Color highlightColor = SRGBA<float>(.99, .89, 0.22, 1.0);
```

Comment:
It's a little hard to figure out what this `SRGBA` object is supposed to represent as opposed to the one above where the names make it easier to see what it's supposed to be the color for the find highlight. Could we somehow make this one a little clearer?